src/services/mock/checkpoint.js

```diff
@@ -0,0 +1,154 @@
+            "owner": "cylc",
+            "host": "eld668.cmpd1.metoffice.gov.uk",
+            "port": 43092,
+            "taskProxies": [
+                {
+                    "id": "cylc/one/20000101T0000Z/sleepy",
+                    "state": "succeeded",
+                    "cyclePoint": "20000101T0000Z",
+                    "task": {
+                        "name": "sleepy"
+                    },
+                    "jobs": [
+                        {
+                            "id": "cylc/one/20000101T0000Z/sleepy/01",
+                            "state": "succeeded",
+                            "submitNum": 1
+                        }
+                    ]
+                },
+                {
+                    "id": "cylc/one/20000101T0000Z/waiting",
+                    "state": "succeeded",
+                    "cyclePoint": "20000101T0000Z",
+                    "task": {
+                        "name": "waiting"
+                    },
+                    "jobs": [
+                        {
+                            "id": "cylc/one/20000101T0000Z/waiting/01",
+                            "state": "succeeded",
+                            "submitNum": 1
+                        }
+                    ]
+                },
+                {
+                    "id": "cylc/one/20000102T0000Z/checkpoint",
+                    "state": "running",
+                    "cyclePoint": "20000102T0000Z",
+                    "task": {
+                        "name": "checkpoint"
+                    },
+                    "jobs": [
+                        {
+                            "id": "cylc/one/20000102T0000Z/checkpoint/01",
+                            "state": "running",
+                            "submitNum": 1
+                        }
+                    ]
+                },
+                {
+                    "id": "cylc/one/20000102T0000Z/sleepy",
+                    "state": "held",
+                    "cyclePoint": "20000102T0000Z",
+                    "task": {
+                        "name": "sleepy"
+                    },
+                    "jobs": []
+                },
+                {
+                    "id": "cylc/one/20000102T0000Z/waiting",
+                    "state": "waiting",
+                    "cyclePoint": "20000102T0000Z",
+                    "task": {
+                        "name": "waiting"
+                    },
+                    "jobs": []
+                },
+                {
+                    "id": "cylc/one/20000102T0000Z/eventually_succeeded",
+                    "state": "succeeded",
+                    "cyclePoint": "20000102T0000Z",
+                    "task": {
+                        "name": "eventually_succeeded"
+                    },
+                    "jobs": [
+                        {
+                            "id": "cylc/one/20000102T0000Z/eventually_succeeded/01",
+                            "state": "running",
```

Comment:
FYI: Something funny is going on at the Cylc Flow end with failed jobs reporting back as running.